src/Logging/Logging.AzureAppServices/src/Microsoft.Extensions.Logging.AzureAppServices.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Logger implementation to support Azure App Services 'Diagnostics logs' and 'Log stream' features.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Did we land somewhere on projects that target only netstandard, or was that just an issue in 3.1?

Reply:
Requirement to target `netcoreappX.Y` exist only for projects in the shared framework